### PR TITLE
chore: ui alignment for value-list and pick-list

### PR DESCRIPTION
**Pull request by asangma**
**Related Issue:** #902

## Summary
Aligned UI of
* PickList
* PickListItem
* ValueList
* ValueListItem

### CODE REVIEW

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss [hunk 1]
```diff
@@ -72,9 +69,13 @@
...
-  align-items: center;
+  align-items: stretch;
   display: flex;
   flex: 0 0 auto;
   justify-self: flex-end;
-  margin: var(--calcite-spacing-quarter) var(--calcite-spacing-half);
+  margin: 0;
+}
+
+.action slot[name="secondary-action"]::slotted(calcite-action) {
```
Comment (driskull) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:79]: This one is a bit complex if we could simplify the selector. If not, no worries.

**Comment by jcfranco** [comment]
Minor, but can you use `chore` as the commit type for calcite-app-component updates until https://github.com/Esri/calcite-components/pull/903 is merged?

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss (hunk 1 above)
Comment (asangma) [reply src/components/calcite-pick-list-item/calcite-pick-list-item.scss:79]: I'll see if it works without the `.action`.

**Comment by asangma** [comment]
@jcfranco Oh yah. Good call.

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss (hunk 1 above)
Comment (asangma) [reply src/components/calcite-pick-list-item/calcite-pick-list-item.scss]: Turns out, I didn't even need that block.

**Comment by asangma** [comment]
Unless there are any udder requests, I'll be MURGLING!

**Pull request merged**